src/styles/blocks/gallery.scss

```diff
@@ -0,0 +1,57 @@
+.gallery {
+  margin-bottom: -40px;
+
+  &__line {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    gap: 0 20px;
+
+    &--1 {
+      grid-template-columns: 2fr 1fr;
+    }
+
+    &--2 {
+      grid-template-columns: 1fr 1fr;
```

Comment:
As I've written earlier - you can get rid of this modifier if you make `grid-template-columns: 1fr 1fr;` by default for your `gallery__line`